test/e2e/framework/pod/utils.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const InfiniteSleepCommand = "trap exit TERM; while true; do sleep 1; done"
+const InfiniteSleepCommandWithoutGracefulShutdown = "sleep infinity"
```

Comment:
Can you add comments to both which explains how they differ and where that matters?

Reply:
@pohly updated.